job-templates/kubernetes_release_1_14.json

```diff
@@ -42,7 +42,7 @@
       "windowsPublisher": "MicrosoftWindowsServer",
```

Comment:
I think we can delete `job-templates/kubernetes_release_1_14.json`. It's not being used anywhere since it's deprecated.

Reply:
1.15 too probably i thik

Reply:
1.15 is still being used. I will submit a PR to remove it from test-infra then we can update this repo to remove it

Reply:
removed 1.15